docker/entrypoint.sh

```diff
@@ -27,12 +27,27 @@ fi
 dockerize -template /etc/temporal/config/config_template.yaml:/etc/temporal/config/docker.yaml
 
 # Automatically setup Temporal Server (databases, Elasticsearch, default namespace) if "autosetup" is passed as an argument.
-for arg; do [[ ${arg} == autosetup ]] && /etc/temporal/auto-setup.sh && break ; done
```

Comment:
`/etc/temporal/auto-setup.sh && break` unintentionally swallowed the failure exit codes from `auto-setup.sh`.